<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="pageWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="SettingsTab" name="settingsPage"/>
      <widget class="LogTab" name="logPage"/>
      <widget class="AboutTab" name="aboutPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionLog"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/gear.ico</normaloff>:/icons/gear.ico</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/document.ico</normaloff>:/icons/document.ico</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/info.ico</normaloff>:/icons/info.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SettingsTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AboutTab</class>
   <extends>QWidget</extends>
   <header>src/window.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
